src/components/select/select.js

```diff
@@ -1276,7 +1276,7 @@ function SelectProvider($$interimElementProvider) {
      * Calculate the
      */
     function calculateMenuPositions(scope, element, opts) {
-      var 
+      var
```

Comment:
It wasn't a question.. i know you can aggregate var declarations into one.

I was asking you to separate it or make the following var declarations (after the semi-colon) aggregated too.